src/core/a-animation.js

```diff
@@ -326,7 +326,7 @@ module.exports.AAnimation = registerElement('a-animation', {
      * Works the same as component mixins but reimplemented because animations
      * aren't components.
      */
-    applyMixin: {
+    handleMixinUpdate: {
```

Comment:
We can discuss about removing `a-animation`. I believe that should be solved by 3rd parties anyways.